modules/stanford_person_importer/stanford_person_importer.post_update.php

```diff
@@ -29,3 +29,52 @@ function stanford_person_importer_post_update_8001(&$sandbox) {
       ->execute();
   }
 }
+
+/**
+ * Delete imported images that are only 1px x 1px.
+ */
+function stanford_person_importer_post_update_8002(&$sandbox) {
+  if (!\Drupal::database()
+    ->schema()
+    ->tableExists('migrate_map_su_stanford_person')) {
+    return;
+  }
+  $media_storage = \Drupal::entityTypeManager()->getStorage('media');
+  $mids = $media_storage->getQuery()
+    ->condition('bundle', 'image')
+    ->condition('field_media_image.width', 2, '<')
+    ->condition('field_media_image.height', 2, '<')
+    ->execute();
+  if (empty($mids)) {
+    return;
+  }
+  $node_storage = \Drupal::entityTypeManager()->getStorage('node');
+  $nids = $node_storage->getQuery()
+    ->condition('su_person_photo', $mids, 'IN')
+    ->execute();
+  if (empty($nids)) {
+    return;
+  }
+
+  /** @var \Drupal\node\NodeInterface $node */
+  foreach ($node_storage->loadMultiple($nids) as $node) {
+    $field_default = $node->getFieldDefinition('su_person_photo')
+      ->getDefaultValue($node);
+    $node->set('su_person_photo', $field_default)->save();
+  }
```

Comment:
Instead of replacing the file with the default, what about leaving it empty and running cron?

Reply:
I can't remember if it populates. But we'd have to save the entity with an empty value. Might as well save it with the default value IMO.